src/fix_engine_state.ml

```diff
@@ -162,10 +162,21 @@ let init_fix_engine_state = {
 }
 ;;
 
+(** THEOREM: one_step calls on busy state eventually terminate *)
+let engine_state_busy engine = 
```

Comment:
The `engine_state_busy` is used in the server loop. I'd like to test the server a little bit more before merging it into master.

The "THEOREM" comment is something that I'd like to be proven (as a VG) about the fix_engine in the future.